resources/views/inc/stats.blade.php

```diff
@@ -0,0 +1,105 @@
+<?php
+$mname = date("F", mktime(0, 0, 0, $month, 1, $year));
+if ($month == 1) { $pm = 12; $pyr = $year - 1; } else { $pm = $month -1; $pyr = $year; }
+if ($month == 12) { $nm = 1; $nyr = $year + 1; } else { $nm = $month + 1; $nyr = $year; }
+?>
+
+<div class="container">
+    <div class="row">
+        <div class="col-sm-4 text-center">
+            <h3>Total Hours this Month</h3>
+            <div class="card text-center" style="background-color:#d3d3d3">
+                <br>
+                <h4>{{ number_format($all_stats['month'], 2) }}</h4>
+                <br>
+            </div>
+        </div>
+        <div class="col-sm-4 text-center">
+            <h3>Total Hours this Year</h3>
+            <div class="card text-center" style="background-color:#d3d3d3">
+                <br>
+                <h4>{{ number_format($all_stats['year'], 2) }}</h4>
+                <br>
+            </div>
+        </div>
+        <div class="col-sm-4 text-center">
+            <h3>Total Hours All Time</h3>
+            <div class="card text-center" style="background-color:#d3d3d3">
+                <br>
+                <h4>{{ number_format($all_stats['total'], 2) }}</h4>
+                <br>
+            </div>
+        </div>
+    </div>
+    <hr>
+    <div class="row">
+        <div class="col-sm-2">
+            <a class="btn btn-primary" href="/controllers/stats/<?=$pyr?>/<?=$pm?>"><i class="fa fa-arrow-left"></i> Previous Month</a></li>
+        </div>
+        <div class="col-sm-8 text-center">
+            <h4>Showing Stats for <?=$mname?> 20<?=$year?></h4>
```

Comment:
Same as the Roster Purge, can we change this to per quarter? Since it's now tracked per quarter. Or, maybe have a toggle to switch between Quarterly and Monthly statistics?

Reply:
I've adjusted these views per the guidance that I received by the DATM. The stats views now have a monthly column (controllers like to see their monthly stats) and a quarterly column.